charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -113,6 +113,7 @@ tolerations: []
 affinity: {}
 
 trustDomain: example.org
+serviceDnsDomain: cluster.local
```

Comment:
DnsDomain sounds a bit like duplicate naming.

Domain Name Service Domain 🤔 

Maybe just `clusterDomain`?

Reply:
Fair point.

Maybe add a documentation comment with that explanation so it ends up in the docs.

Reply:
Ok, I can get behind that. I didn't see anything in the helm chart but the spire-controller-manager's config having an option for it makes sense. I'll fix it up.

Reply:
We havent added it to the helm chart yet. Not sure its been released yet, it was just merged a couple weeks ago.